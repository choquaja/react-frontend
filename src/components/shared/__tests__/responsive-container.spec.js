import React from 'react';
import { shallow } from 'enzyme';
import ResponsiveContainer from '../responsive-container';

describe('responsive container', () => {
  it('renders without crashing', () => {
    shallow(<ResponsiveContainer />);
  });

  it('renders with fluid prop', () => {
    const component = shallow(<ResponsiveContainer fluid />);
    expect(component).toMatchSnapshot();
  });
});
