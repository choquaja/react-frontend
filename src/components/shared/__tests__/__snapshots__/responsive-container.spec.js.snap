// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`responsive container renders with fluid prop 1`] = `
<div
  className="responsive-container__ResponsiveContainer-dGkraX cPOYWa"
/>
`;
